libpod/runtime.go

```diff
@@ -263,6 +263,12 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 	configPath := ConfigPath
 	foundConfig := true
 	if rootless.IsRootless() {
+		if runtime.config.SignaturePolicyPath == "" {
+			newPath := filepath.Join(os.Getenv("HOME"), ".config/containers/policy.json")
```

Comment:
Might be better to do a single lookup for $HOME and use that both here and below

Reply:
Not specific to this PR, but XDG_CONFIG_HOME should be honored

Reply:
yes good idea, pushed an updated version